### PR TITLE
[SRE-126] promtail added to manifest.yaml

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**

### CODE REVIEW

File: cmd/downloader/downloader/downloader.go [hunk 1]
```diff
@@ -78,40 +78,51 @@ func DownloadService(flags types.CliFlags, manifest *types.BoxManifest, service
...
+			outputPath = filepath.Join(extractPath, service.OutputPath+".exe")
+		}
 	}
 	zipReader, err := zip.OpenReader(archiveFile)
 	if err != nil {
 		return err
 	}
 	for _, file := range zipReader.File {
-		if strings.HasSuffix(file.Name, service.ArchivePath) {
+		if strings.HasPrefix(file.Name, service.ArchivePath) {
```
[the remark this answers is not in the document]
Comment (hjpotter92) [reply cmd/downloader/downloader/downloader.go:125]: what about updating the tarball extract function with same logic? would keep it consistent for future edits too!

[the remark this answers is not in the document]
Comment (pwilczynskiclearcode) [reply cmd/downloader/downloader/downloader.go]: I think I need to implement regexp... as neither suffix not prefix matches all of our cases

[the remark this answers is not in the document]
Comment (hjpotter92) [reply cmd/downloader/downloader/downloader.go]: > I think I need to implement regexp... as neither suffix not prefix matches all of our cases

oh, what about updating the manifest structure a little bit? something like: ```yaml
archivePath:
  prefix: <optional>
  suffix: <optional>
  regexp: <optional>
```
wdyt? if `regexp` is provided, other 2 are ignored.

[the remark this answers is not in the document]
Comment (pwilczynskiclearcode) [reply cmd/downloader/downloader/downloader.go]: I'd go with `regexp` only as it covers all of the cases.

**Comment by pwilczynskiclearcode** [comment]
> The O's mainly care about how our Bs are behaving and reading a subset of logs from our Bs. They are somewhat interested in their own Bs pushing to the same loki instance but this is not a priority for the time being and something they're willing to explore later.
https://discord.com/channels/423160867534929930/1118181111793713172/1119135865965400074

**Pull request closed without merging**